elements/lisk-transactions/src/sign_multi_signature_transaction.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import * as cryptography from '@liskhq/lisk-cryptography';
+
+import { DelegateTransaction } from './10_delegate_transaction';
+import { VoteTransaction } from './11_vote_transaction';
+import { MultisignatureTransaction } from './12_multisignature_transaction';
+import { TransferTransaction } from './8_transfer_transaction';
+import { BaseTransaction } from './base_transaction';
+import { TransactionJSON } from './transaction_types';
+
+export interface SignatureObject {
+	readonly publicKey: string;
+	readonly signature: string;
+	readonly transactionId: string;
+}
+
+// tslint:disable-next-line no-any
+const transactionMap: { readonly [key: number]: any } = {
```

Comment:
```suggestion
const transactionMap: { readonly [key: number]: typeof BaseTransaction } = {
```

Reply:
For some reason this is failing. Resolving for now to complete the PR maybe open an issue to check in different issue or as tech debt?